test/framework/kogitoserviceutils.go

```diff
@@ -105,7 +106,6 @@ func newImageOrDefault(fullImage string, defaultImageName string) v1alpha1.Image
 
 		image.Domain = config.GetServicesImageRegistry()
 		image.Namespace = config.GetServicesImageNamespace()
-		image.Name = defaultImageName
```

Comment:
Is this correct? If the GetServicesImageNameSuffic is not provided, then the name won't be set.

Reply:
Yes, if the name is not set then default name is applied by Kogito operator.
So we don't have to specify it by ourselves but rather leverage operator functionality where possible.

Reply:
Ok, thanks for explaining!

Reply:
it turns out that this is needed for CLI installer, returning it back